README.md

```diff
@@ -69,15 +69,31 @@ fixed -> BinaryType
 ```
 
 As for unions, we only support three kinds of unions:
+
 1) union(int, long)
+
 2) union(float, double)
+
 3) union(something, null), where something is one of the avro types mentioned above, including two types of unions.
 
 At the moment we ignore docs, aliases and other properties present in the avro file.
 
+## Supported types for SparkSQL -> Avro conversion
+
+Every SparkSQL type is supported. For most of them the corresponding type is obvious (e.g. IntegerType gets converted to int), for the rest the following conversions are used:
+
+```
+ByteType -> int
+ShortType -> int
+DecimalType -> string
+BinaryType -> bytes
+TimestampType -> long
+StructType -> record
+```
+
 ### Scala API
 
-You can use the library by loading the implicits from `com.databricks.spark.avro._`.
+To get SchemaRDD from avro file, you can use the library by loading the implicits from `com.databricks.spark.avro._`.
```

Comment:
nit: to get *a* SchemaRDD (also, this will have to be updated once the data frame change goes in, just as a side note)